ksql-common/src/main/java/io/confluent/ksql/util/SchemaUtil.java

```diff
@@ -49,14 +49,14 @@
   private static final Map<Type, Supplier<Schema>> typeToSchema
       = ImmutableMap.<Type, Supplier<Schema>>builder()
       .put(String.class, () -> Schema.OPTIONAL_STRING_SCHEMA)
-      .put(boolean.class, () -> Schema.OPTIONAL_BOOLEAN_SCHEMA)
+      .put(boolean.class, () -> Schema.BOOLEAN_SCHEMA)
```

Comment:
this makes a lot of sense.

Might want to check with Hojjat this doesn't break something else though.